terraform_env/prod/outputs.tf

```diff
@@ -0,0 +1,12 @@
+output "webapp_url" {
+    value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "cd_webhook" {
```

Comment:
Worth noting that this should really be marked sensitive as well - if you look at the values interpolated in it has a password!